frontend/web/components/feature-page/FeatureFilters.tsx

```diff
@@ -0,0 +1,221 @@
+  search: '',
+  sort: { label: 'Name', sortBy: 'name', sortOrder: 'asc' },
+  tag_strategy: 'INTERSECTION',
+  tags: [],
+  value_search: '',
+}
+
+const sortToHeader = (s: any) => {
+  if (!s) return DEFAULTS.sort
+  if ('sortBy' in s) return s
+  return {
+    label: s.label || DEFAULTS.sort.label,
+    sortBy: s.value || DEFAULTS.sort.sortBy,
+    sortOrder: DEFAULTS.sort.sortOrder,
+  } as FiltersValue['sort']
+}
+
+// Converts filters to url params, excluding ones that are already default
+export function getURLParamsFromFilters(f: FiltersValue) {
+  const existing = Utils.fromParam() as Record<string, string | undefined>
+
+  return {
+    ...existing,
+    group_owners: f.group_owners?.length ? f.group_owners.join(',') : undefined,
+    is_archived: f.is_archived ? 'true' : undefined,
+    is_enabled:
+      f.is_enabled === null ? undefined : f.is_enabled ? 'true' : 'false',
+    owners: f.owners?.length ? f.owners.join(',') : undefined,
+    page: f.page !== DEFAULTS.page ? String(f.page) : undefined,
+    search: f.search || undefined,
+    tag_strategy:
+      f.tag_strategy !== DEFAULTS.tag_strategy
+        ? String(f.tag_strategy)
+        : undefined,
+    tags: f.tags?.length ? f.tags.join(',') : undefined,
+    value_search: f.value_search || undefined,
+  }
+}
+// Gets expected filters from URL parameters
+export const getFiltersFromURLParams = (
+  params: Record<string, string | undefined>,
+) => {
+  return {
+    group_owners:
+      typeof params.group_owners === 'string'
+        ? params.group_owners.split(',').map((v) => parseInt(v))
+        : [],
+    is_archived: params.is_archived === 'true',
+    is_enabled:
+      params.is_enabled === 'true'
+        ? true
+        : params.is_enabled === 'false'
+        ? false
+        : null,
+    owners:
+      typeof params.owners === 'string'
+        ? params.owners.split(',').map((v) => parseInt(v))
+        : [],
+    page: params.page ? parseInt(params.page) - 1 : 1,
+    releasePipelines: [],
+    search: params.search || '',
+    sort: {
+      label: Format.camelCase(params.sortBy || 'Name'),
+      sortBy: params.sortBy || 'name',
+      sortOrder: params.sortOrder || 'asc',
+    },
+    tag_strategy: params.tag_strategy || 'INTERSECTION',
+    tags:
+      typeof params.tags === 'string'
+        ? params.tags.split(',').map((v) => parseInt(v))
+        : [],
+    value_search:
+      typeof params.value_search === 'string' ? params.value_search : '',
+  } as FiltersValue
+}
+
+//Converts filter to api expected properties
+export const getServerFilter = (f: FiltersValue) => ({
+  ...f,
+  group_owners: f.group_owners?.length ? f.group_owners : undefined,
+  owners: f.owners.length ? f.owners : undefined,
+  search: (f.search || '').trim(),
+  sort: sortToHeader(f.sort),
+  tags: f.tags.length ? f.tags.join(',') : undefined,
+})
+
+//Detect if the filter is default
+const isDefault = (v: FiltersValue) =>
+  isEqual(
+    {
+      ...v,
+    },
```

Comment:
Do we need to spread? isEqual seems sufficient no ? 
```suggestion
v,
```